plugins/custom_caret.lua

```diff
@@ -28,11 +27,11 @@ local config = require "core.config"
 local DocView = require "core.docview"
 
 config.plugins.custom_caret = common.merge({
-    shape = style.custom_caret.shape,
-    color_r = style.custom_caret.color[1],
-    color_g = style.custom_caret.color[2],
-    color_b = style.custom_caret.color[3],
-    opacity = style.custom_caret.opacity,
+    shape = style.caret_shape,
```

Comment:
Since `style.caret_shape` isn't available by default on the litexl style this value should be the string `"line"`, is better to use a plugin config option than assuming the user will modify the style.